Bot failure

There was a problem with the **Conda nightly tests** workflow, please investigate.

Last night's [nightly](https://github.com/mamba-org/mamba/actions/runs/6682657255/job/18158079953#step:9:992) failed 12 tests:

```
=========================== short test summary info ============================
FAILED mamba/tests/test_all.py::test_update[bash] - AssertionError: assert '2.0.7' == '2.0.0'
  - 2.0.0
  ?     ^
  + 2.0.7
  ?     ^
FAILED mamba/tests/test_all.py::test_track_features[bash] - IndexError: list index out of range
FAILED mamba/tests/test_all.py::test_create_dry_run[True-True] - json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
FAILED mamba/tests/test_all.py::test_create_dry_run[True-False] - json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
FAILED mamba/tests/test_all.py::test_create_dry_run[False-False] - AssertionError: assert 'Total download' in 'Currently, only install, create, list, search, run, info, clean, remove, update, repoquery, activate and deactivate are supported through mamba.\n'
FAILED mamba/tests/test_all.py::test_create_files - json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
FAILED mamba/tests/test_all.py::test_multi_channels[config_file0] - json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
FAILED mamba/tests/test_all.py::test_update_py - AssertionError: assert '3.8' in '3.10.13 | packaged by conda-forge | (main, Oct 26 2023, 18:07:37) [GCC 12.3.0]'
FAILED mamba/tests/test_all.py::test_unicode - json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
FAILED mamba/tests/test_all.py::test_info[True] - json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
FAILED mamba/tests/test_all.py::test_info[False] - AssertionError: assert ('mamba version : ' + '1.5.1') in 'Currently, only install, create, list, search, run, info, clean, remove, update, repoquery, activate and deactivate are supported through mamba.\n'
FAILED mamba/tests/test_all.py::test_remove_virtual_package[__linux] - AssertionError: assert '__linux' in 'Currently, only install, create, list, search, run, info, clean, remove, update, repoquery, activate and deactivate are supported through mamba.'
 +  where 'Currently, only install, create, list, search, run, info, clean, remove, update, repoquery, activate and deactivate are supported through mamba.' = <built-in method join of str object at 0x7f3d1d7e3cf0>(['Currently, only install, create, list, search, run, info, clean, remove, update, repoquery, activate and deactivate are supported through mamba.'])
 +    where <built-in method join of str object at 0x7f3d1d7e3cf0> = '\n'.join
============= 12 failed, 11 passed, 2 skipped in 119.28s (0:01:59) =============

```

--- a/.github/workflows/conda_canary.yml
+++ b/.github/workflows/conda_canary.yml
@@ -28,7 +28,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-        ref: '1.x'  # Conda latest is on 1.x version
+        with:
+          ref: '1.x'  # Conda latest is on 1.x version
       - name: create mamba build environment
         uses: mamba-org/setup-micromamba@v1
         with:

--- a/.github/workflows/conda_canary.yml
+++ b/.github/workflows/conda_canary.yml
@@ -27,7 +27,8 @@ jobs:
   conda_nightly:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
+        ref: '1.x'  # Conda latest is on 1.x version
       - name: create mamba build environment
         uses: mamba-org/setup-micromamba@v1
         with:

